### PR TITLE
Adds powerbi third-party tool

### PR DESCRIPTION
**Pull request by goodroot**

### CODE REVIEW

**Comment by goodroot** [comment]
https://questdb.io/docs/reference/api/postgres/

File: documentation/third-party-tools/powerbi.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+4. Enter your QuestDB connection details:
+   - Server: `localhost` (or your server address)
+   - Database: `qdb`
+   - Data Connectivity mode: `Import`
+   - Advanced options (optional):
+     - Port: `8812` (default QuestDB PGWire port)
+     - Command timeout: Adjust based on your query complexity
+
+5. Select either:
+   - Windows authentication (if configured)
```
Comment (jerrinot) [inline documentation/third-party-tools/powerbi.md:33]: When would you like to choose this? Is it working at all? Some Active Directory magic? Does QuestDB support it for PGWire at all?

File: documentation/third-party-tools/powerbi.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    avg(price) avg_price
+FROM trades
+WHERE timestamp >= dateadd('M', -1, now())
+SAMPLE BY 1d
+ALIGN TO CALENDAR;
+```
+
+## Best Practices
+
+- Leverage [timestamps](/docs/guides/working-with-timestamps-timezones/ functions for time-series analysis
```
Comment (jerrinot) [inline documentation/third-party-tools/powerbi.md:88]: a bad link, missing `)`

File: documentation/third-party-tools/powerbi.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+This guide demonstrates how to connect QuestDB with Microsoft PowerBI to create
+interactive data visualizations and dashboards.
+
+## Prerequisites
+
+- [QuestDB](/docs/quick-start) running locally or remotely
+- [PowerBI Desktop](https://powerbi.microsoft.com/) installed
+
+## Connection Setup
```
Comment (jerrinot) [inline documentation/third-party-tools/powerbi.md:14]: I think a screenshot would go a long way here. for the connection setup.

Comment (goodroot) [reply documentation/third-party-tools/powerbi.md:14]: The PowerBI has images that are borrow-able. 

I have no current means to take proper screens.

**Pull request merged**